pyproject.toml

```diff
@@ -73,7 +73,10 @@ envlist = site
 
 [testenv]
 deps=
+  pytest-asyncio
   pytest-cov
+  pytest-timeout
+  pytest-xdist
```

Comment:
This looks like a rebase error -- that looks like it should be in the "add tox support" commit, not in the "tweak version number scheme" one?

Reply:
Thanks for the catch.  `xdist` also doesn't really belong in this list — we don't actually use it.